gossip/src/duplicate_shred_handler.rs

```diff
@@ -40,10 +43,15 @@ pub struct DuplicateShredHandler {
     blockstore: Arc<Blockstore>,
     leader_schedule_cache: Arc<LeaderScheduleCache>,
     bank_forks: Arc<RwLock<BankForks>>,
+    epoch_schedule: EpochSchedule,
```

Comment:
If we are adding this field, then we probably don't need `cached_slots_in_epoch` anymore.